src/test/java/org/opensearch/ad/e2e/DetectionResultEvalutationIT.java

```diff
@@ -563,8 +565,7 @@ public void testValidationWindowDelayRecommendation() throws Exception {
             .extractValue("model", responseMap);
         // adding plus one since window delay always rounds up another minute
         assertEquals(
-            String
-                .format(Locale.ROOT, CommonErrorMessages.WINDOW_DELAY_REC, +recDetectorIntervalMinutes + 1, recDetectorIntervalMinutes + 1),
+            String.format(Locale.ROOT, CommonErrorMessages.WINDOW_DELAY_REC, minutes + 1, minutes + 1),
```

Comment:
Just to confirm - this means that before it was hardcoded to a 3 min, and expected an error message suggesting 4 min. And now it's dynamically based on the latest timestamp of the ingested data, which if it takes 1 min to ingest, would be set to 4 min, and expect an error message suggesting 5 min?

Reply:
Yes the hardcoded 3 minutes will always be there as its the delay of the first point in the data set and ingestion time is added now like in your example.

Reply:
Could you add comments to explain why you use minutes + 1 ?

Reply:
I have comment stating: `// adding plus one since window delay always rounds up another minute` since we round up in window delay recommendation, should I add more to this?

Reply:
I'll just change to `(long) Math.ceil(expectedWindowDelayMillis / 60000.0)` which has same effect and is cleaner

Reply:
This is enough. Thanks.